composer.json

```diff
@@ -18,6 +18,7 @@
         "league/oauth2-client": "~1.0",
         "league/oauth2-facebook": "dev-master",
         "league/oauth2-google": "dev-master",
+        "rtheunissen/oauth2-reddit": "~1.0@dev",
```

Comment:
I don't want to do that. It's very common for packages to have a different namespace and package name. Take a look at almost any of the third part providers.